projects/plugins/jetpack/tests/e2e/lib/utils-helper.js

```diff
@@ -160,6 +160,31 @@ async function logAccessLog() {
 	logger.slack( { type: 'debuglog', message: apacheLog } );
 }
 
+/**
+ * Formats a given file name by replacing unaccepted characters (e.g. space)
+ *
+ * @param {string} filePath the file path. can be absolute file path, file name only, with or without extension
+ * @param {boolean} includeTimestamp if true, the current timestamp will be added as a prefix
+ * @return {string} the formatted file path
+ */
+function fileNameFormatter( filePath, includeTimestamp = true ) {
+	const parts = path.parse( path.normalize( filePath ) );
+	let fileName = parts.name;
+	const ext = parts.ext;
+	const dirname = parts.dir;
+
+	if ( includeTimestamp ) {
+		const now = new Date();
+		const formattedDate = `${ now.getFullYear() }${ now.getMonth() }${ now.getDay() }${ now.getHours() }${ now.getMinutes() }${ now.getSeconds() }${ now.getMilliseconds() }`;
+
+		fileName = `${ formattedDate }_${ fileName }`;
```

Comment:
Since now we can name files properly, I think the `fileName` is more important than a date.
```suggestion
		fileName = `${ fileName }_${ formattedDate }`;
```

Reply:
I preferred having the timestamp first so we can have the videos in order. A relevant example is the Mailchimp test, which at some point will open a WP.com page in a new tab. This page will be recorded, but because it is opened without an event name, the video name will only have the timestamp. If the timestamp is first in the name we'll have this video exactly after the main one and we can easily identify it:

```
202122162836684_Paid_blocks___MailChimp_Block.webm
202122162855312__.webm
```

Reply:
That makes sense. My motivation has based on the fact, that in most cases I am interested in a specific video file. With the current structure, I would need to resize my Finder window to spot it :) But I'm fine with that, at least for now 😜

Reply:
I definitely know what you mean, I too had to make this big sacrifice and resize my Finder 😆 .